{"name": "test_openanaccount_a1_error_4", "status": "passed", "description": "账户开户失败的用例", "attachments": [{"name": "log", "source": "6cebae1b-c472-4c0f-989c-6a5e5016e168-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "800c02e9-07eb-47f2-acc8-3fb9e45195b3-attachment.txt", "type": "text/plain"}], "start": 1617349380917, "stop": 1617349383207, "uuid": "5ae2a528-fa63-4aa5-bedc-fa6e15a8c8bd", "historyId": "6fcf15fae37fda2eed735daccb323105", "testCaseId": "046ab45e8173c2672dd98df412457b29", "fullName": "tests.test_a8_the_customer_account.TestTheCustomerAccount#test_openanaccount_a1_error_4", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_a8_the_customer_account"}, {"name": "subSuite", "value": "TestTheCustomerAccount"}, {"name": "host", "value": "DESKTOP-U13POE8"}, {"name": "thread", "value": "6480-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_a8_the_customer_account"}]}